conf/test.config

```diff
@@ -1,43 +1,234 @@
+        params.dbsnp               = params.test_data['homo_sapiens']['genome']['dbsnp_138_hg38_21_vcf_gz']
+        params.known_indels        = params.test_data['homo_sapiens']['genome']['mills_and_1000g_indels_21_vcf_gz']
+        params.fasta               = params.test_data['homo_sapiens']['genome']['genome_21_fasta']
+        params.intervals           = params.test_data['homo_sapiens']['genome']['genome_21_multi_interval_bed']
+        params.step                = 'variant_calling'
+        params.wes                 = true
+
+        params.nucleotides_per_second = 20
+    }
+    tools_tumoronly {
+        params.input               = "${projectDir}/tests/csv/3.0/recalibrated_tumoronly.csv"
+        params.dbsnp               = params.test_data['homo_sapiens']['genome']['dbsnp_138_hg38_21_vcf_gz']
+        params.fasta               = params.test_data['homo_sapiens']['genome']['genome_21_fasta']
+        params.germline_resource   = params.test_data['homo_sapiens']['genome']['gnomad_r2_1_1_21_vcf_gz']
+        params.intervals           = params.test_data['homo_sapiens']['genome']['genome_21_multi_interval_bed']
+        params.pon                 = params.test_data['homo_sapiens']['genome']['mills_and_1000g_indels_21_vcf_gz']
+        params.step                = 'variant_calling'
+        params.wes                 = true
+
+        params.nucleotides_per_second = 20
+    }
+    tools_somatic {
+        params.input               = "${projectDir}/tests/csv/3.0/recalibrated_somatic.csv"
+        params.chr_dir             = params.test_data['homo_sapiens']['genome']['genome_21_chromosomes_dir']
+        params.dbsnp               = params.test_data['homo_sapiens']['genome']['dbsnp_138_hg38_21_vcf_gz']
+        params.fasta               = params.test_data['homo_sapiens']['genome']['genome_21_fasta']
+        params.germline_resource   = params.test_data['homo_sapiens']['genome']['gnomad_r2_1_1_21_vcf_gz']
+        params.intervals           = params.test_data['homo_sapiens']['genome']['genome_21_multi_interval_bed']
+        params.pon                 = params.test_data['homo_sapiens']['genome']['mills_and_1000g_indels_21_vcf_gz']
+        params.step                = 'variant_calling'
+        params.wes                 = true
+
+        params.nucleotides_per_second = 20
+    }
+    // can only be tested locally due to too large cram files for GHA
+    // download corresponding input files (ascat_somatic.csv) from ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data/
+    // test works only without chromosome annotated loci files available at https://github.com/VanLoo-lab/ascat/tree/master/ReferenceFiles/WGS
+    tools_somatic_ascat{
+        params.input               = "${baseDir}/tests/csv/3.0/ascat_somatic.csv"
+        params.chr_dir             = params.test_data['homo_sapiens']['genome']['genome_21_chromosomes_dir']
+        params.ascat_loci          = "/mnt/volume/repos/modules/test_ascat2/G1000_loci_hg19.zip"
```

Comment:
Is there anyway not to have a such a path here?
@SusiJo 
Can't we store that in test-datasets@sarek?

Reply:
We could add the chromosome based loci maybe also to igenomes